java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -2323,6 +2323,37 @@ public static ColumnView makeStructView(ColumnView... columns) {
     return makeStructView(columns[0].rows, columns);
   }
 
+  /**
+   * Create a column of bool values indicating whether the specified scalar
+   * is an element of each row of a list column.
+   * Output `column[i]` is set to null if one or more of the following are true:
+   * 1. The key is null
+   * 2. The column vector list value is null
+   * 3. The list row does not contain the key, and contains at least
+   *    one null.
+   * @param key the scalar to look up
+   * @return a Boolean ColumnVector with the result of the lookup
+   */
+  public final ColumnVector listContains(Scalar key) {
+    assert type.equals(DType.LIST) : "column type must be a LIST";
```

Comment:
Should we add a test that the `Scalar` type matches with the `ColumnVector`'s element type?

Reply:
Will add an assert I think that is a good point!

Reply:
We should check for the column column version too , isnt it?

Reply:
Did for both and it looks a tad bit ugly but let me know if this works :)

Reply:
Fair warning - I havent tested it yet but we should be fine IMO